gym/core.py

```diff
@@ -43,7 +43,7 @@ def _deprecate_mode(render_func):  # type: ignore
         def render(
             self: object, *args: Tuple[Any], **kwargs: Dict[str, Any]
         ) -> render_return:
-            if "mode" in kwargs.keys():
+            if "mode" in kwargs.keys() or len(args) > 0:
```

Comment:
Should any `kwargs` or `args` be passed to render?

Reply:
Why do we not check `if len(kwargs) > 0 or len(args) > 0:`?

Reply:
Oh, I see what you mean.

For example, right now, no deprecation warning with 
```
env.render(width=400, height=400)
```

This is more related to the new render API; do we want to forbid any type of kwarg for the render method?

Reply:
For the environment that I worked on, no kwarg to render is allowable. Nevertheless, for other environments that generate rendering at the end, it can be useful to allow it. 
I would let this flexibility